### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by patricoferris**

### CODE REVIEW

**Comment by patricoferris** [comment]
HOW DO I ADD ANOTHER FILE VIA THE GITHUB BROWSER API!

**Comment by patricoferris** [comment]
Ok I managed

**Comment by RyanGibb** [comment]
> HOW DO I ADD ANOTHER FILE VIA THE GITHUB BROWSER API!

WHY ARE YOU USING THE GITHUB BROWSER API

File: static/index.md [hunk 1]
```diff
@@ -63,6 +79,6 @@ Roll(s): _Poppy Seed_
 
 Description: _Initially, very broccoli-forward based on just the smell. This flipped to being very thai-green-curry-forward (lemongrass, thai basil, garlic etc.) after tasting._
 
-Rating: **6/10**
+Rating: **4/10**
```
Comment (RyanGibb) [inline static/index.md:82]: :(

**Pull request merged**